pipelines/storage_network/Jenkinsfile

```diff
@@ -23,6 +25,7 @@ node {
 
     withCredentials([
         usernamePassword(credentialsId: CREDS_ID, passwordVariable: 'AWS_SECRET_KEY', usernameVariable: 'AWS_ACCESS_KEY'),
+        string(credentialsId: 'DO_CREDS', variable: 'DO_TOKEN'),
```

Comment:
We should not use digital ocean resources anymore. Let's remove them used in the current test code in another PR.

Reply:
Removed in https://github.com/longhorn/longhorn-tests/pull/1775.